bpf/Makefile

```diff
@@ -24,7 +24,7 @@ BPF_BUNDLE := $(OUT_DIR)/parca-agent.bpf.tar.gz
 LIBBPF_HEADERS := $(OUT_DIR)/libbpf/$(ARCH)/usr/include
 
 VMLINUX_INCLUDE_PATH := $(SHORT_ARCH)
-BPF_SRC := unwinders/native.bpf.c unwinders/go_runtime.h
+BPF_SRC := unwinders/native.bpf.c unwinders/go_runtime.h unwinders/hash.h unwinders/common.h unwinders/shared.h
```

Comment:
Maybe this should be 

`BPF_SRC := unwinders/native.bpf.c $BPF_HEADERS`

And we should move BPF_HEADERS up.  Or the way the rules are written looks like BPF_SRC doesn't need to include the headers.